packages/fxa-shared/l10n/branding.ftl

```diff
@@ -44,8 +45,10 @@
 -product-firefox-relay = Firefox Relay
 -product-pocket = Pocket
 
+-brand-apple = Apple
+-brand-google = Google
 -brand-paypal = PayPal
+-brand-stripe = Stripe
```

Comment:
I like consistency as much as the next person, but it'd be much more efficient/safer to just move the term as `-brand-name-stripe` since this term doesn't already exist in branding.ftl so we could keep strings and terms as-is, and not have to trigger re-translation of the term.